examples/contrib/gp/sv-dkl.py

```diff
@@ -87,10 +105,13 @@ def main(args):
                                    shuffle=True)
     test_loader = get_data_loader(dataset_name='MNIST',
                                   data_dir=data_dir,
-                                  batch_size=args.batch_size,
+                                  batch_size=args.test_batch_size,
                                   dataset_transforms=[transforms.Normalize((0.1307,), (0.3081,))],
                                   is_training_set=False,
-                                  shuffle=True)
+                                  shuffle=False)
+    if args.cuda:
+        train_loader.num_workers = 1
+        test_loader.num_workers = 1
```

Comment:
this is to train faster in GPU (18s vs 22s for each epoch)

Reply:
Do you think this might fix https://github.com/uber/pyro/issues/1540 ? Should we set both to 1 always, to avoid hanging on CI?

Reply:
@fehiepsi - The default value of num_workers is 0, which corresponds to not invoking multiprocessing at all. Did you try with a higher number for `num_workers` say 2, or 4? 

@fritzo - By default, pytorch's dataloader does not invoke multiprocessing, so we should be good. @fehiepsi has already isolated the cause for #1540 which is platform dependent flaky regression `nn.conv2D` (probably related - https://github.com/pytorch/pytorch/issues/14456).

Reply:
@neerajprad I thought that it is just useful to load data while GPU is doing its stuff so didnt try a higher number. Will check it tonight. :) In MNIST pytorch example, they set numworker=1 and pin memory is True when using GPU. I don’t see any difference with pin memory so I don’t put it here.
@fritzo I isolated the bug at 
https://travis-ci.org/fehiepsi/travis_test/builds/467326092
with just a simple cnn operator. In some specific Travis system, it is fast. In the other ones it is slow.

Reply:
@neerajprad I set a higher num workers but it does not help for speed

Reply:
Np, I was only curious if it might. Looks good!